eo-runtime/src/test/java/org/eolang/AtCompositeTest.java

```diff
@@ -34,6 +34,7 @@
  *
  * @since 0.16
  */
+@SuppressWarnings("PMD.JUnit5TestShouldBePackagePrivate")
```

Comment:
@yegor256 Temporarily this class should be public because it stores
```java
public static final String TO_ADD_MESSAGE = "TO ADD ASSERTION MESSAGE";
```
There is todo about it: https://github.com/c71n93/eo/blob/88b3b4bf5e7cf54efd3a116aac7a41cc35ce54ef/eo-runtime/src/test/java/org/eolang/AtCompositeTest.java#L42. This decision was made here https://github.com/objectionary/eo/pull/3113#pullrequestreview-2011200831